daffodil-core/src/main/scala/org/apache/daffodil/grammar/BitOrderMixin.scala

```diff
@@ -45,34 +45,36 @@ trait BitOrderMixin extends GrammarMixin with ByteOrderAnalysisMixin { self: Ter
    *
    * If uncertain, returns false.
    */
-  final protected lazy val isKnownSameBitOrder: Boolean = {
-    val res =
-      if (enclosingTerm.isEmpty) false // root needs bit order
-      else {
-        val optPrior = this.nearestPriorPhysicalTermSatisfying(_.optDefaultBitOrder.isDefined)
-        optPrior match {
-          case None => false // no prior that has a bit order we could be the same as
-          case Some(prior) => {
-            if (prior.defaultBitOrder =:= this.defaultBitOrder) true
-            else false
-          }
-        }
-      }
-    res
-  }
+  final protected lazy val isKnownSameBitOrder: Boolean = false // FIXME: must analyze prior terms, enclosing terms, etc.
```

Comment:
Missed this. Need to investigate if this disables an important optimization. This file was converted to stubs and needs to be reconsidered.